cypress/integration/test_table.ts

```diff
@@ -37,10 +37,12 @@ describe("Tests Table", () => {
     });
   });
 
-  it("Should display No Data when given an invalid TaskID in the url", () => {
+  it("Should display error banner when given an invalid TaskID in the url", () => {
```

Comment:
nit. `Should display **a** error banner when given a

Reply:
this is also a personal preference. there does not need to be an 'a'